Editor/NeedsRuntimeConversion/Extensions/UrdfRobotExtensions.cs

```diff
@@ -63,34 +63,36 @@ public static void Create(string filename, ImportSettings settings)
             Undo.RegisterCreatedObjectUndo(robotGameObject, "Create " + robotGameObject.name);
             Selection.activeObject = robotGameObject;
 
-            CorrectAxis(robotGameObject, settings.choosenAxis);
+            CorrectAxis(robotGameObject);
             CreateCollisionExceptions(robot, robotGameObject);
         }
 
-        public static void CorrectAxis(GameObject robot, ImportSettings.axisType axis = ImportSettings.axisType.yAxis)
+        public static void CorrectAxis(GameObject robot)
         {
-            UrdfVisual[] visualMeshList = robot.GetComponentsInChildren<UrdfVisual>();
-            UrdfCollision[] collisionMeshList = robot.GetComponentsInChildren<UrdfCollision>();
             UrdfRobot robotScript = robot.GetComponent<UrdfRobot>();
+            if (robotScript.CheckOrientation())
+                return;
+            Quaternion correctYtoZ = Quaternion.Euler(-90, 0, 90);
+            Quaternion correctionZtoY = Quaternion.Inverse((correctYtoZ));
```

Comment:
nit: shall we keep names consistent? like: correctionZtoY/correctZtoY, or correctYtoZ/correctionYtoZ